openfisca_france_local/aides_jeunes_reform.py

```diff
@@ -69,7 +69,7 @@ def is_department_eligible(individu: Population, period: Period, parameters: Par
     return sum(eligibilities) > 0
```

Comment:
Du coup avec mypy j'ai cette erreur qui est relevée : 

```
Incompatible return value type (got "Union[ndarray[Any, dtype[bool_]], bool]", expected "ndarray[Any, Any]")Mypy[return-value](https://mypy.readthedocs.io/en/latest/_refs.html#code-return-value)
(variable) eligibilities: list
```

ET j'ai testé cette solution qui a l'air de fonctionner; qu'en penses-tu ?

```suggestion
return any(eligibilites)
```

Reply:
J'ai passé pas mal de temps sur ce warning, et effectivement il est possible de le faire disparaître avec ta solution.
Ce qui me cause de soucis c'est pourquoi cette fonction en particulier provoque ce warning.
Je n'arrive pas à comprendre ce qui change par rapports autres fonctions, pour moi c'est la même chose.

Du coup, je me dis que changer ce code va amener de la confusion à la lecture (puisqu'on perd en cohérence) mais que je ne sais pas ce qu'on gagne...

Reply:
L'origine du problème a l'air de provenir de l'utilisation de `startswith` qui renvoie un ` NDArray[bool_]` tandis que dans la méthode `is_region_eligible` on utilise `region == TypesCodeInseeRegion(code_region)` qui renvoie des `bool`

Reply:
J'ai retourner le code un peu dans tous les sens, essayé avec différentes valeurs, je n'arrive pas à faire en sorte d'avoir une valeur de retour qui vaut un simple booléen.

Si tu veux on peur regarder ensemble?